### PR TITLE
Feature/increasing masks capacity

### PR DESCRIPTION
**Pull request by MurashovVen**
- Creating Uint128 type and writing unit tests for it.
- Changing upload and download masks type on Uint128.

### CODE REVIEW

**Comment by stewartie4** [comment]
Very happy that this change is being made - 32 blobber limit was very limiting.
However why have a complicated mask with binary shifting? Why not just have a limit and loop through the list of blobbers conventionally?

**Comment by Sriep** [comment]
How should this interact with PR https://github.com/0chain/gosdk/pull/22 ?

**Comment by stewartie4** [comment]
> How should this interact with PR https://github.com/0chain/gosdk/pull/22 ? 
> 
> Are the unit tests from PR22 still good?

https://github.com/0chain/gosdk/pull/22 Should merge before this change as it is agnostic to the actual limit in practice 
Feel free to merge it

File: zboxcore/sdk/allocation.go [hunk 1]
```diff
@@ -474,7 +473,7 @@ func (a *Allocation) downloadFile(localPath string, remotePath string, contentMo
 	downloadReq.localpath = localPath
 	downloadReq.remotefilepath = remotePath
 	downloadReq.statusCallback = status
-	downloadReq.downloadMask = ((1 << uint32(len(a.Blobbers))) - 1)
+	downloadReq.downloadMask = zboxutil.NewUint128(1).Lsh(uint64(len(a.Blobbers))).Sub64(1)
```
Comment (stewartie4) [inline zboxcore/sdk/allocation.go:476]: See upload mask comment

**Pull request merged**